app/components/contact-item/modal.js

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isBlank } from '@ember/utils';
+
+export default class ContactItemModalComponent extends Component {
+  @service currentSession;
+  @service store;
+
+  @tracked contact;
```

Comment:
Deze heb je niet meer nodig. Je kan overal `this.args.contact` gebruiken (en `@contact` in de template)